src/app-interoperability/application-package-definition.linkml.yaml

```
@@ -0,0 +1,497 @@
+          so it is possible to present a user interface with a logical grouping of the parameters in each section. 
+          See the [Section](#section-attributes) section below.
+        rank: 10
+        range: Section
+        multivalued: true
+        required: true
+      schema:
+        description: >-
+          Schema is used to provide details about how to validate each parameter value. 
+          At a minimum, the parameter value must be validated to match the schema's data type. 
+          The schema indicates additional rules the provided value must satisfy to be considered valid input. 
+          See the [Schema](#schema-attributes) section below.
+        rank: 20
+        range: Schema
+        multivalued: true
+        required: true
+
+  Section:
+    description: Named sections within the configuration layout.
+    rank: 120
+    attributes:
+      name:
+        description: >-
+          The name of the section. This may be used in the user interface to show the grouping of the associated parameters within the section.
+        rank: 10
+        range: string
+        required: true
+      settings:
+        description: >-
+          Settings are used to provide instructions to the workload orchestration software vendor for displaying parameters to the user. 
+          A user MUST be able to provide values for all settings. 
+          See the [Setting](#setting-attributes) section below.
+        rank: 20
+        range: Setting
+        multivalued: true
+        required: true
+
+  Setting:
+    description: Individual configuration settings.
+    rank: 130
+    attributes:
+      parameter:
+        description: The name of the [parameter](#parameter-attributes) the setting is associated with.
+        rank: 10
+        range: string
+        required: true
+      name:
+        description: The parameter's display name to show in the user interface.
+        rank: 20
+        range: string
+        required: true
+      description:
+        description: The parameters's short description to provide additional context to the user in the user interface about what the parameter is for.
+        rank: 30
+        range: string
+      immutable:
+        description: If true, indicates the parameter value MUST not be changed once it has been set and used to install the application. Default is false if not provided.
+        rank: 40
+        range: boolean
+      schema:
+        description: The name of the schema definition to use to validate the parameter's value. See the [Schema](#schema-attributes) section below.
+        rank: 50
+        range: string
+        required: true
+
+  Schema:
+    description: Validation rules for parameter values.
+    rank: 140
+    attributes:
+      name:
+        description: The name of the schema rule. This used in the [setting](#setting-attribute) to link the setting to the schema rule.
+        rank: 10
+        range: string
+        required: true
+      datatype:
```

@arne-broering typo (capital 'T' in the middle of datatype)

```suggestion
      dataType:
```